data/migrations/20181011143118_instructions.js

```diff
@@ -0,0 +1,20 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('instructions', tbl =>{
+        tbl.increments();
+         tbl
+        .integer('step_number')
+        .notNullable()
+         tbl
+        .string('step')
+        .notNullable()
```

Comment:
Create a separate table for `steps`, and use this table to connect each step to a `recipe`, like a many-to-many relationship.